src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -98,7 +98,7 @@ internal void CalculateEffectivePackageSaveMode()
             {
                 try
                 {
-                    var xDocument = XDocument.Load(projectConfigFilePath);
+                    var xDocument = Common.XmlUtility.Load(projectConfigFilePath);
```

Comment:
Don't use var per [Rule 10](https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style)

Reply:
Can we clean up the imports?

Specifically why can't we just say 

```cs
XmlUtility.Load(projectConfigFilePath);
```

Reply:
> Can we clean up the imports?

Compiler complained about ambiguous references between `NuGet.Common.XmlUtility` and `NuGet.Packaging.XmlUtility`. Hence I added this extra namespace qualifier. I can also specify an using statement to resolve the ambiguity. `using XmlUtility = NuGet.Common.XmlUtility;`.